### PR TITLE
Voron 1.x/2.x hepa filter exhaust

### PR DESCRIPTION
**Pull request by marcdebruijn**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
A HEPA filter replacement for the stock exhaust assembly.

#### How was it tested? 
Tested in my lab (aka, my dining room) for 40 hrs. It definitely makes a difference, especially when coupled with a Nevermore filter. This also works in tandem with the nevermore to create a slight negative pressure in the build chamber to prevent leaking through cracks.

#### Screenshots (if appropriate)
in the images directory- will add more at a later time.

### CODE REVIEW

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: printer_mods/README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ like so:
 |DaGrooch13 |[Tophat Exhaust Filter V0](./DaGrooch13/Tophat_Exhaust_Filter) |Activated charcoal exhaust filter with HEPA that replaces the rear tophat panel |:heavy_check_mark: |:x: |:x: |:x:|
 |Dan88 |[V0 one-part Skirts](./Dan88/V0skirt) |One part Skirts for the V0 |:heavy_check_mark: |:x: |:x: |:x:|
 |DC67|[Plug Panel Mod for Schaffner FN286-10-06](./DC67/plug_panel_schaffner)|A modified plug panel for the Power Entry Connector Schaffner FN286-10-06 with two integreated Keystone sockets. |:x:|:grey_question:|:heavy_check_mark:| :x: |
+|dePrintinator |[HEPA Exhaust](./dePrintinator/HEPA_Filter_Exhaust)| HEPA filter chamber exhaust replacement. |:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|
```
[the remark this answers is not in the document]
Comment (marcdebruijn) [reply printer_mods/README.md]: Fixed.

File: printer_mods/dePrintinator/HEPA_Filter_Exhaust/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+![HEPA Chamber Exhaust](./Images/hepa-chamber-exhaust-cad.png)
+
+
+## <B>Bill of Materials</B>
+
+### <B>New Parts</B>
+
+- 1x 120x120x25mm fan such as [Noctua NF-F12 iPPC-2000](https://www.amazon.com/gp/product/B00KFCQT6M/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1). (note that I have an adjustable voltage on my control board, but make sure your fan matches the voltage for your system.)
+- HEPA Filter such as [Nispira True HEPA Filter Replacement with Activated Carbon](https://www.amazon.com/gp/product/B0921DZLPK/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1)
```
[the remark this answers is not in the document]
Comment (marcdebruijn) [reply printer_mods/dePrintinator/HEPA_Filter_Exhaust/README.md]: Fixed.

**Review by cbc02009 — APPROVED** [review]

**Comment by cbc02009** [comment]
Looks like the CI check found some STL errors that need fixing.

**Comment by marcdebruijn** [comment]
> Looks like the CI check found some STL errors that need fixing.

Roger that- fixed the non-manifold geometry issue, and hopefully it should pass the CI pipeline now.

**Pull request merged**